mm2src/coins_activation/src/tendermint_token_activation.rs

```diff
@@ -68,11 +68,16 @@ impl TokenActivationOps for TendermintToken {
         let my_address = token.my_address()?;
         let mut balances = HashMap::new();
         balances.insert(my_address, balance);
+
         let init_result = TendermintTokenInitResult {
             balances,
             platform_coin: token.platform_ticker().into(),
         };
 
+        token
+            .platform_coin
+            .add_activated_token_info(token.ticker.clone(), token.decimals, token.denom.clone());
+
```

Comment:
As I can see, this is not implemented in `erc20_token_activation` and `slp_token_activation`. We already have `RegisterTokenInfo` trait for it, it should be used here:
https://github.com/KomodoPlatform/atomicDEX-API/blob/b2f1eb112ec8834ef7c637d59a2e03f18672b4cb/mm2src/coins_activation/src/token.rs#L133-L135

Code:
```rust
platform_coin.register_token_info(&token);
```